src/main/java/com/cloudbees/jenkins/support/impl/JVMProcessSystemMetricsContents.java

```diff
@@ -47,22 +48,25 @@ public boolean isSelectedByDefault() {
         }
     }
 
-    static final Map<String, String> UNIX_PROC_CONTENTS;
+    static final Set<ProcFile> UNIX_PROC_CONTENTS;
```

Comment:
I don't think it has compatibility implications but why this change?

Reply:
Because now you can decide whether to filter or not for every file. The new class ```ProcFile``` let you do that. Instead of just add the file and filter it. But I think this change is a mistake, because I had to have created a new ```AdvancedJVMProcessSystemMetricsContents``` and leave this class as it was. I've searched for **UNIX_PROC_CONTENTS** along GitHub and there are no occurrences of this outside of the plugin.

Reply:
So do you pretend to create the new class or leave it as it is?

Reply:
My thought is because it's not used anywhere and because it has the feature of the previous class, we can leave it so